contentcuration/contentcuration/frontend/channelList/views/Channel/StudioMyChannels.vue

```diff
@@ -0,0 +1,537 @@
+    <div class="my-channels__new-channel">
+      <KButton
+        v-if="isEditable && !loading"
+        primary
+        data-test="add-channel"
+        :text="$tr('channel')"
+        @click="newChannel"
+      />
+    </div>
+    <div class="my-channels__body">
+      <p
+        v-if="listChannels && !listChannels.length"
+        class="my-channels__body--no-channels"
+      >
+        {{ $tr('noChannelsFound') }}
+      </p>
+      <KCardGrid
+        layout="1-1-1"
+        :loading="loading"
+        class="my-channels__cards"
+      >
+        <KCard
+          v-for="(channel, index) in listChannels"
+          :key="channel.id"
+          :headingLevel="2"
+          thumbnailDisplay="small"
+          :thumbnailSrc="thumbnailSrc(channel)"
+          :thumbnailAlign="'left'"
+          :thumbnailScaleType="'fitXY'"
+          :orientation="windowIsSmall ? 'vertical' : 'horizontal'"
+          :title="channel.name"
+          :titleMaxLines="2"
+          :data-testid="`card-${index}`"
+          @click="goToChannelRoute(channel)"
+        >
+          <template #belowTitle>
+            <div class="my-channels__cards--below-title">
+              <div class="my-channels__cards--below-title__resource">
+                <span> {{ $tr('resourceCount', { count: channel.count || 0 }) }} </span>
+                <span>
+                  {{ language(channel) }}
+                </span>
+              </div>
+              <div class="my-channels__cards--below-title__desc">{{ channel.description }}</div>
+            </div>
+          </template>
+          <template #footer>
+            <div class="my-channels__cards--footer">
+              <div class="my-channels__cards--footer__left">
+                <span v-if="channel.last_published">
+                  {{
+                    $tr('lastPublished', {
+                      last_published: $formatRelative(channel.last_published, { now: new Date() }),
+                    })
+                  }}
+                </span>
+                <span v-else>
+                  {{ $tr('unpublishedText') }}
+                </span>
+                <div>
+                  <KTooltip
+                    :reference="`lastUpdatedTime-${index}`"
+                    placement="bottom"
+                    :refs="$refs"
+                  >
+                    {{
+                      $tr('lastUpdated', {
+                        updated: $formatRelative(channel.modified, { now: new Date() }),
+                      })
+                    }}
+                  </KTooltip>
+                  <KIcon
+                    v-if="hasUnpublishedChanges(channel)"
+                    :ref="`lastUpdatedTime-${index}`"
+                    :color="$themePalette.green.v_600"
+                    icon="dot"
+                  />
+                </div>
+              </div>
+              <div class="my-channels__cards--footer__right">
+                <KRouterLink
+                  v-if="!libraryMode"
```

Comment:
Alongside similar comments, `libraryMode` will never be truthy on `StudioMyChannels` so it can be removed from here and elsewhere in the file (+ I've just learned that it's an obsolete mode that doesn't take effect in the current version either)